ws-server/src/server.rs

```diff
@@ -241,7 +241,13 @@ async fn background_task(
 	while !stop_monitor.shutdown_requested() {
 		data.clear();
 
-		method_executors.select_with(receiver.receive_data(&mut data)).await?;
+		if let Err(e) = method_executors.select_with(receiver.receive_data(&mut data)).await {
```

Comment:
NOTE; this will kill a connection on any soketto::connection::Error,

In a followup PR I think we should try to do some graceful error handling